utils/ansible/ftest/templates/daos-make.sh.j2

```diff
@@ -0,0 +1,209 @@
+#!/bin/bash
+
+set -o pipefail
+
+VERSION=0.1
+CWD="$(realpath $(dirname $0))"
+
+DAOS_INSTALL_DIR="{{ daos_runtime_dir }}/install"
+DAOS_SOURCE_DIR="{{ daos_source_dir }}"
+SERVERS_LIST="{{ groups["daos_servers"] | join(',') }}"
+{% if "daos_clients" in groups and groups["daos_clients"] | length > 0 %}
```

Comment:
(lint) You need a space after the '{'. [SC1054]
(lint) This } is literal. Check expression (missing ;/\n?) or quote it. [SC1083]